libs/template/presets/storefront/experience/footer.ts

```diff
@@ -34,11 +34,30 @@ const legalLinks: ExperienceComponent = {
       type: 'oryx-content-text',
       content: { data: { text: `©️ 2023 Spryker` } },
     },
-    link('Imprint', '/faq/imprint'),
-    link('Terms & conditions', '/article/terms-and-conditions'),
-    link('Privacy Notice', '/article/privacy'),
-    link('Data preference'),
-    link('Condition of use'),
+    ...(featureVersion >= '1.4'
+      ? [
+          {
+            type: 'oryx-content-list',
+            options: {
+              tags: 'legal',
+              rules: [
+                {
+                  layout: {
+                    divider: true,
+                  },
+                  style: 'display: contents;',
+                },
+              ],
+            },
+          },
+        ]
+      : [
+          link('Imprint', '/faq/imprint'),
+          link('Terms & conditions', '/article/terms-and-conditions'),
+          link('Privacy Notice', '/article/privacy'),
+          link('Data preference'),
+          link('Condition of use'),
+        ]),
```

Comment:
We should keep the footer as-is (at least for now), and then merge the CMS driven content in using labs.